src/hooks/useRefs/useRefs.ts

```diff
@@ -6,7 +6,11 @@ import { createRef, useMemo, type RefObject } from 'react';
 export function useRefs<T extends Record<string | symbol, RefObject<unknown>>>(
   initialTarget?: Partial<T>,
 ): T {
-  const proxyTarget = useMemo(() => initialTarget ?? {}, [initialTarget]);
+  const proxyTarget = useMemo(
```

Comment:
If the `useMemo` doesn't have dependencies, it means the `proxyTarget` never changes.

If the `proxyTarget` never changes, shouldn't it be a `useRef` instead?

Which means that the `new Proxy` can also be a `useRef`, and the `.current` can be returned from this hook.